server/tests/models/tagArticle.spec.js

```diff
@@ -27,46 +21,83 @@ before(async () => {
   }
 });
 
-describe('Tag Article relationship', async () => {
+describe('Tag Article relationship', () => {
+  before(async () => {
+    try {
+      const newUserId = await getUserId('testarticle@email.com', 'testarticle2');
+      const newCategoryId = await getCategoryId('technology');
+      articleData = getArticleData(newArticle, { userId: newUserId, categoryId: newCategoryId });
+      article = await Article.create(articleData);
+    } catch (error) {
+      console.log(error);
+    }
+  });
+
   it('should tag an article', async () => {
-    const newUserId = await getUserId('testarticle@email.com', 'testarticle2');
-    const newCategoryId = await getCategoryId('technology');
-    articleData = getArticleData(newArticle, { userId: newUserId, categoryId: newCategoryId });
-    article = await Article.create(articleData);
-    articleId = article.id;
-    tagId = await getTagId('EPIC');
-    const tagArticles = await article.addTags(tagId);
-    const tagArticleId = tagArticles[0].dataValues.ArticleId;
-    const tagArticleTagId = tagArticles[0].dataValues.TagId;
-
-    expect(tagArticleId).to.be.equal(articleId);
-    expect(tagArticleTagId).to.be.equal(tagId);
+    try {
+      articleId = article.id;
+      tagId = await getTagId('EPIC');
+      const tagArticles = await article.addTags(tagId);
+      const tagArticleId = tagArticles[0].dataValues.ArticleId;
+      const tagArticleTagId = tagArticles[0].dataValues.TagId;
+
+      expect(tagArticleId).to.be.equal(articleId);
```

Comment:
Could the assertions be outside of the try-catch block, so that mocha can pick up the error chai raises? i.e `try{ awaits }
                    catch(error){ error handling}
                      (assertions)`